logger/logger.go

```diff
@@ -54,15 +54,28 @@ func SetLogger(zl *zap.Logger) {
 	logger = &Logger{zl.Sugar()}
 }
 
+const JSONLogFileName = "log.jsonl"
+
+// ProductionLoggerFilepath returns the full path to the file the
+// ProductionLogger logs to.
+func ProductionLoggerFilepath(configRootDir string) string {
+	return path.Join(configRootDir, JSONLogFileName)
+}
+
 // CreateProductionLogger returns a log config for the passed directory
 // with the given LogLevel and customizes stdout for pretty printing.
-func CreateProductionLogger(dir string, jsonStdout bool, lvl zapcore.Level) *zap.Logger {
+func CreateProductionLogger(
+	dir string, jsonStdout bool, lvl zapcore.Level, toDisk bool) *zap.Logger {
 	config := zap.NewProductionConfig()
-	destination := path.Join(dir, "log.jsonl")
 	if !jsonStdout {
-		config.OutputPaths = []string{"pretty", destination}
+		config.OutputPaths = []string{"pretty"}
+	}
+	config.ErrorOutputPaths = []string{"stderr"}
```

Comment:
You can remove this line since it is redundant to the default.

Reply:
Ah, in `zap/config.go`. Got it. Thanks.